dist/tools/bmp/bmp.py

```diff
@@ -258,9 +304,9 @@ def parse_args():
     parser.add_argument('--tpwr', action='store_true', help='enable target power')
     parser.add_argument('--serial', help='choose specific probe by serial number')
     parser.add_argument('--port', help='choose specific probe by port (overrides auto selection)')
-    parser.add_argument('--attach', help='choose specific target by number', default='1')
+    parser.add_argument('--attach', help='choose specific target by number', type=int, default='1')
```

Comment:
```suggestion
    parser.add_argument('--attach', help='choose specific target by number', type=int, default=1)
```

As the type is `int`, we can provide the default also as `int`.

Reply:
Added a fixup commit to address this.